joomla/joomla-cms
# [Code Style] changes for Joomla 2.5 SQL file

If you export a live site's database and compare it to the core joomla.sql you'll see lots of minor differences. This hardens the database examination and shows that joomla.sql contains many code style mistakes.

**What I did:**
- Removed the spacing errors;
- Names for all variables are now of the same case.
- Database tables are now listed in alphabetical order;  <<< changes removed

The last issue caused most of problems, because the tables arrangement in the original joomla.sql is not alphabetical. Many of the tables are listed almost randomly, which makes it difficult to compare your site database with the original joomla.sql.

**How to test:**
To make sure that my changes are correct you can do the following:

1) Get an empty database;
Import the original joomla.sql file;
Export the current database to the file e.g. joomla-var1.sql

2) Drop all tables in the database;
Import my joomla.sql file;
Export the current database to the file e.g. joomla-var2.sql

3) Compare joomla-var1.sql and joomla-var2.sql - there shouldn't any differences between them.
